pkg/services/cloudmigration/api/api.go

```diff
@@ -202,10 +195,14 @@ type RunCloudMigrationRequest struct {
 // 403: forbiddenError
 // 500: internalServerError
 func (cma *CloudMigrationAPI) GetMigrationRun(c *contextmodel.ReqContext) response.Response {
+	// TODO Parameter 'uid' is not actually used at the moment, since we just need the run UID.
```

Comment:
check comment in PR description about this

Reply:
Decided to take care of it here so we can stop messing with this :D